contrib/sqlalchemy-connector/setup.py

```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/python3
+# This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+import os
+import re
+import sys
+
+from setuptools import find_packages
+from setuptools import setup
+
+v = open(
+    os.path.join(
+        os.path.dirname(os.path.realpath(sys.argv[0])), "sqlalchemy_starrocks", "__init__.py"
+    )
+)
+VERSION = re.compile(r".*__version__ = \"(.*?)\"", re.S).match(v.read()).group(1)
+v.close()
+
+
+setup(
+    name="sqlalchemy_starrocks",
```

Comment:
If following the code style of the python package, it should be sqlalchemy-starrocks